vm_test.go

```diff
@@ -149,9 +149,9 @@ func TestHyperRelocationNewcontainer(t *testing.T) {
 	err = vm.relocateHyperCommand(cmd)
 	assert.NotNil(t, err)
 
-	vm.Close()
```

Comment:
Seems logical for @sameo to have the OpenSockets and CloseSockets being symmetrical, meaning we consider a case where you want to CloseSockets before you shutdown the VM as something wrong.
I'll update the patch based on the latest virtcontainers vendoring (including the patch for CloseSockets()), feel free to merge it if you are okay with this. Otherwise, tell me what you think as an alternative.